src/NodaTime.Web/wwwroot/css/site.css

```diff
@@ -260,29 +265,27 @@ footer .design {
   font-style: italic;
 }
 
-.accordion footer {
-  background-color: #fff;
-  padding: 10px 16px;
-  font-size: 12px;
-  border: 0px none;
+.side-nav {
+  padding-bottom: 0;
 }
 
-.accordion-navigation {
+.side-nav > .heading {
   background-color: #FFE76B;
   padding: 10px;
+  margin-bottom: 0;
 }
 
-.accordion > .content {
-  background-color: #B17250;
+.side-nav > .content {
+  background-color: #5c72db;
+  margin-bottom: 0;
 }
 
-.accordion .side-nav > li > a,
-.accordion .side-nav > li > a:visited,
-.accordion .side-nav > li > a:hover,
-.accordion .side-nav > li > a:focus {
-  color: #FFF;
-  padding-top: 0;
-  padding-bottom: 0;
+.side-nav > li > a,
+.side-nav > li > a:not(.button),
```

Comment:
I think this may need to be e.g. `a:not(.button):focus` (etc) in order to have the same priority as the Foundation-default rules it's overriding. At present, we get a light-blue text colour.

Reply:
This was to override just `a:not(.button)` - without that, it looked awful, though I can't remember in what way. I'm happy with the light blue for the value we're hovering over, unless that doesn't have enough contrast.